### ISSUE TITLE
when using the package in the project it does not see its files!

### ISSUE THREAD

**Issue by esracoskun** [hosseinmd/react-signalr#18]
Hello,

The package usage and version are as I mentioned below, but it gives the following error.
Can you help.

Error
```
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/example/node_modules/react-signalr/src/webSocket/utils.ts' file: Error: ENOENT: no such file or directory, open '/example/node_modules/react-signalr/src/webSocket/utils.ts'
```

Use

package.json 
`"react-signalr": "^0.2.14",`

App.js
```
import {
  createSignalRContext, // SignalR
  createWebSocketContext, // WebSocket
  createSocketIoContext, // Socket.io
} from 'react-signalr'

const { token } = StoreAuthentication.useState();

const SignalRContext = createSignalRContext()

<SignalRContext.Provider
    connectEnabled={!!token}
    accessTokenFactory={() => token}
    dependencies={[token]}
    url={'https://example/hub'}
  >
    <NavigationScroll>
      <Routes />
    </NavigationScroll>
</SignalRContext.Provider>
```

**Comment by hosseinmd** [comment]
This is a warning. Don't worry about it. I will fix it.

**Comment by esracoskun** [comment]
While only https://example/hub is given in the url, why it adds https://example/hub/negotiate?negotiateVersion=1 after the request.

**Comment by hosseinmd** [comment]
It's signalR internal's rout, don't worry about it

**Comment by hosseinmd** [comment]
Fixed: v0.2.15
